src/components/v2/Table/Head.tsx

```diff
@@ -24,7 +24,7 @@ function Head<C extends { key: string; label: string; orderable: boolean }[]>({
   onRequestOrder,
   className,
 }: IHeadProps<C>) {
-  const styles = useStyles();
+  const styles = useStyles({});
```

Comment:
I'd make the param optional instead. Passing an empty object doesn't make sense

Reply:
done